net.hachune.mikutter.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: net.hachune.mikutter
+runtime: org.gnome.Platform
+runtime-version: 3.28
+sdk: org.gnome.Sdk
+command: mikutter
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --socket=pulseaudio
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+  - --persist=.mikutter
```

Comment:
Ideally it should use $XDG_CONFIG_DIR (something to report to upstream).

Reply:
Using `$XDG_CONFIG_DIR` is disscussed in upstream, but it seems difficult task (migration of existing configuration, separate cache between multiple instances, etc.). TBD

Upstream ticket: https://dev.mikutter.hachune.net/issues/1189